app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -313,6 +355,14 @@ def ca_cert
 
     private
 
+    def os_name_mapping(host)
+      host.operatingsystem.name =~ /redhat|centos/i ? 'rhel': host.operatingsystem.name.downcase
+    end
```

Comment:
`rhel` should be ok, as that's wat we translate to here, I guess `Red Hat could be hanelded. What I wanted to cover mainly is the form the puppet uses